namara/__init__.py

```diff
@@ -14,21 +15,24 @@ def __init__(self, api_key, debug=False, host='https://api.namara.io', api_versi
         self.base_path = '{0}/{1}'.format(self.host, self.api_version)
         self.headers = {'Content-Type': 'application/json', 'X-API-Key': api_key}
     
-    def export_file_to_dataframe(self, dataset_id, organization_id, project_id, options=None):
-        list_of_chunks = [] 
+    @retry(wait=wait.wait_exponential(multiplier=1, min=4, max=10), stop=stop.stop_after_attempt(5))
```

Comment:
Retry isn't necessary and in fact could lead to some other issues.  I would just get rid of this.